Tiltconfig.yaml

```
@@ -118,6 +118,22 @@ dexcom:
     clientURL: "https://sandbox-api.dexcom.com"
 ### Dexcom API Config End ###
 
+### Abbott API Config Start ###
+abbott:
+  secret:
+    enabled: false
+    data_:
+      ClientId: ""
+      ClientSecret: ""
+      StateSalt: ""
+  configmap:
+    enabled: true
+    redirectURL: ""
+    tokenURL: "https://oauth.partner.int.adc-lv.io/oauth2/token"
+    authorizeURL: "https://oauth.partner.int.adc-lv.io/oauth2/auth"
+    clientURL: "https://api.partner.int.adc-lv.io"
```

Why don't we just remove these three URLs from this file and just use ""? We'll just include them with the client id and secret in the 1Password entries for developers to add to their `local/Tiltconfig.yaml`.
```suggestion
    tokenURL: ""
    authorizeURL: ""
    clientURL: ""
```